### PR TITLE
PrusaSlicer: fix .desktop file

### PR DESCRIPTION
**Pull request by ericonr**
- Name used in the template was different from the filename.

### CODE REVIEW

**Comment by q66** [comment]
remove the `broken=` for ARM and add `nocross=yes`

File: srcpkgs/PrusaSlicer/template [hunk 1]
```diff
@@ -16,12 +16,7 @@ license="GPL-3.0-or-later"
 homepage="https://www.prusa3d.com/prusaslicer/"
 distfiles="https://github.com/prusa3d/Prusaslicer/archive/version_${version}.tar.gz"
 checksum=e6e0c83bf92e448ec058fd3063b84caca69f58b8b419e48eace6e8ce534937c0
-
-broken="files/prusa-slicer.desktop is missing"
-
-case "$XBPS_TARGET_MACHINE" in
-	arm*) broken="https://build.voidlinux.org/builders/armv7l_builder/builds/25230/steps/shell_3/logs/stdio" ;;
-esac
+nocross=yes # https://build.voidlinux.org/builders/armv7l_builder/builds/25230/steps/shell_3/logs/stdio
```
Comment (pullmoll) [inline srcpkgs/PrusaSlicer/template:19]: `nocross="https://build.voidlinux.org/builders/armv7l_builder/builds/25230/steps/shell_3/logs/stdio"` works just as well with the advantage of the URL being printed.

Comment (ericonr) [reply srcpkgs/PrusaSlicer/template]: Thanks!

**Comment by q66** [comment]
oh, the revision bump was unnecessary, but whatever

**Pull request merged**